### PR TITLE
Navigator.NavigationBar Landscape Fix

### PR DESCRIPTION
**Pull request by yamill**
NavigationBar items are fixed to Portrait when Device is in Landscape.

This supplements that fix by removing the `width` properties and just using `left` and `right` positioning respectively.

<strong>Before:</strong>

ios simulator screen shot sep 8 2015 1 27 02 pm

<strong>After:</strong>

ios simulator screen shot sep 8 2015 1 29 21 pm

### CODE REVIEW

File: Libraries/CustomComponents/Navigator/NavigatorNavigationBarStyles.ios.js [hunk 1]
```diff
@@ -52,18 +52,16 @@ var BASE_STYLES = {
...
     opacity: 1,
-    width: SCREEN_WIDTH / 3,
     height: NAV_BAR_HEIGHT,
     backgroundColor: 'transparent',
   },
   RightButton: {
     position: 'absolute',
     top: STATUS_BAR_HEIGHT,
-    left: 2 * SCREEN_WIDTH / 3,
+    right: 0,
```
Comment (ide) [inline Libraries/CustomComponents/Navigator/NavigatorNavigationBarStyles.ios.js]: This makes a lot of sense

Comment (yamill) [inline Libraries/CustomComponents/Navigator/NavigatorNavigationBarStyles.ios.js]: :smile:

**Comment by ericvicenti** [comment]
Thank you for fixing this up!

**Comment by yamill** [comment]
Np! Glad to contribute!

**Pull request closed without merging**